boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.yaml

```diff
@@ -26,3 +26,4 @@ supported:
   - usb_device
   - watchdog
   - sdhc
+  - sctimer
```

Comment:
This should be `- pwm` to trigger twister:
https://github.com/zephyrproject-rtos/zephyr/tree/15c77acfb4326b7c39f3d2d48f13fbd9fe30404e/tests/drivers/pwm/pwm_api/src